assets/app/scripts/services/data.js

```diff
@@ -42,7 +42,7 @@ angular.module('openshiftConsole')
     for (var i = 0; i < subAttrs.length; i++) {
       attrValue = attrValue[subAttrs[i]];
       if (!attrValue) {
-        return;
+        attrValue = "";
```

Comment:
missing a `break`? can we add a minimal number of tests for this? maybe one simple, one complex?